atc/worker/client.go

```diff
@@ -278,6 +278,10 @@ func (client *client) chooseTaskWorker(
 			}
 			existingContainer, err = client.pool.ContainerInWorker(logger, owner, containerSpec, workerSpec)
 			if err != nil {
+				release_err := activeTasksLock.Release()
+				if release_err != nil {
+					return nil, release_err
```

Comment:
Hey,

Wdyt of leveraging a `multierror` (e.g., https://github.com/concourse/concourse/blob/da466557298f5c4f146064a80065dca1cd07f9ac/atc/gc/volume_collector.go#L39) here? 

This way we wouldn't lose the original error that came from `client.pool.ContainerInWorker`.

Thanks!

Reply:
Yes! Good point!